### DISCUSSION TITLE
请教一下如何实现这种减字古琴谱的最优化输入?

### DISCUSSION THREAD

**Discussion by fishfree** [rime/home#900] [Ideas]
减字谱（把一些字的偏旁部首组合在一起，形成的字符数约有几千个）：https://zh.wikipedia.org/wiki/%E5%87%8F%E5%AD%97%E8%B0%B1 https://baike.baidu.com/item/%E5%87%8F%E5%AD%97%E8%B0%B1/7109858

我想能尽可能兼顾以下目标：

1. 在别的电脑上最好不要安装字体，但我感觉可能无法避免，那么是否可以做成web font？
2. 放在Unicode编码空间内，这个似乎还要向国际标准组织提交议案讨论审定后才能被采纳。
3. 最好能复用现有拼音输入法，或者做成RIME的插件？

我自己瞎想的思路：把谱里的基础组字偏旁部首做成web字体，并使用拼音输入法输入，然后使用类似于Mathjax这种可以在web中把多个偏旁部首组合渲染成一个古琴谱的字。我一个门外汉，不知道这样是否可行？而且似乎也只考虑了web端，没有考虑到本地环境中如何渲染显示。

还请前辈们多多指教！先谢过了！

**Comment by LEOYoon-Tsaw** [comment]
1. 不可能，字體當然得自己裝，rime又不是網葉，做甚web font，做夢罷
2. 對啊，去找U源提啊，等二十年後可能就能加入了。不過目前可以用IDS，得配合專門的字體顯示
3. 不難
